data/unaligned_labeled_mask_dataset.py

```diff
@@ -69,6 +69,8 @@ def __getitem__(self, index):
 
         try:
             A_img = Image.open(A_img_path).convert('RGB')
+            #if self.input_nc == 1:
+            #    A_img = A_img.convert('L')
```

Comment:
Remove?

Reply:
I was sure you'd see it. If you don't mind, I've left it here, because at the moment there's this RGB conversion (in other dataset types as well actually), and the grayscale transform comes later on. Whereas, we could just transform once... But I've left it as an optimization later.